module/VuFind/src/VuFind/Db/Row/UserList.php

```diff
@@ -54,6 +58,7 @@
 class UserList extends RowGateway implements \VuFind\Db\Table\DbTableAwareInterface, UserListEntityInterface
 {
     use \VuFind\Db\Table\DbTableAwareTrait;
+    use DbServiceAwareTrait;
```

Comment:
You'll need to apply the DbServiceAwareInterface to the class in addition to using the trait. The interface is a marker that the service manager uses to automatically inject dependencies; without it, things probably won't work in some circumstances (though tests won't start failing until we start using the new methods).